src/20230831_AaveV2_Eth_ChaosLabsRiskParameterUpdates/AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831.t.sol

```diff
@@ -0,0 +1,341 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {ProtocolV2TestBase, ReserveConfig} from 'aave-helpers/ProtocolV2TestBase.sol';
+import {AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831} from './AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831.sol';
+
+/**
+ * @dev Test for AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831
+ * command: make test-contract filter=AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831
+ */
+contract AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831_Test is ProtocolV2TestBase {
+  AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18033340);
+    proposal = new AaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831();
+  }
+
+  function testProposalExecutionPartOne() public {
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
+      'preAaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831',
+      AaveV2Ethereum.POOL
+    );
+
+    GovHelpers.executePayload(vm, address(proposal), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'postAaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831',
+      AaveV2Ethereum.POOL
+    );
+
+    validateAssetBatchOne(allConfigsBefore, allConfigsAfter);
+    validateAssetBatchTwo(allConfigsBefore, allConfigsAfter);
+    validateAssetBatchThree(allConfigsBefore, allConfigsAfter);
+
+    diffReports(
+      'preAaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831',
+      'postAaveV2_Ethereum_ChaosLabsRiskParameterUpdates_20230831'
+    );
+
+    e2eTest(AaveV2Ethereum.POOL);
+  }
+
+  function validateAssetBatchOne(
+    ReserveConfig[] memory allConfigsBefore,
+    ReserveConfig[] memory allConfigsAfter
+  ) public pure {
+    // BAL
+    ReserveConfig memory BAL_UNDERLYING_CONFIG = _findReserveConfig(
+      allConfigsBefore,
+      AaveV2EthereumAssets.BAL_UNDERLYING
+    );
+
+    BAL_UNDERLYING_CONFIG.liquidationThreshold = 35_00;
```

Comment:
I think that in the future it makes sense to create constants with all the parameter changes you intend to use to reduce the need to rely on magic values